osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/factories/InvocationCounterManagerTest.java

```diff
@@ -38,9 +39,13 @@
 
   @Mock private DlmsHelper dlmsHelper;
 
+  @Mock private DlmsDeviceRepository deviceRepository;
```

Comment:
The InvocationCounterManager now saves the invocationCounter to this repository. I would expect that the test verifies whether a call to the repository is done with the right value for the counter.